src/supremm/supremmconf.py

```diff
@@ -1,29 +1,41 @@
 #!/usr/bin/env python
 """ helper utiility to print out config info """
+from __future__ import print_function
 
 import sys
 import os
 import json
+import logging
 from getopt import getopt
 
 from supremm.config import Config
+from supremm.scripthelpers import setuplogger
 
 def usage():
     """ print usage """
-    print "usage: {0} [OPTS]".format(os.path.basename(__file__))
-    print "  -s --section SECTION  output the configuration data from the specified section"
-    print "  -i --item ITEM        output the configuration data for the specified item"
-    print "  -h --help             print this help message"
-
+    print("usage: {0} [OPTS]".format(os.path.basename(__file__)))
+    print("  -d --debug            set log level to debug")
+    print("  -c --config           specify the configuration file")
+    print("  -r --resources        output the configuration data for all enabled resources") 
```

Comment:
There doesn't appear to be any code that implements the `-r` flag that is listed here. 

Also the help text for the `-c` option is misleading - you need to specify the path to the directory that contains the conifguration file and not the file path.

Reply:
Updated the misleading text. As for the -r flag, I think I had an idea to list individual resources but forgot about it. Seeing that it's the same as `python supremmconf.py -s resources` I've just removed it.